device-types/Synology/RS1221RP-plus.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Synology
+model: RS1221RP+
+slug: synology-rs1221rp-plus
+part_number: RS1221RP+
+u_height: 2
+is_full_depth: true
+comments: '[Synology RS1221+/RS1221RP+ Datasheet](https://global.synologydownload.com/download/Document/Hardware/DataSheet/RackStation/21-year/RS1221%2B/ger/Synology_RS1221%2B_Data_Sheet_ger.pdf)'
+weight: 8.4
+weight_unit: kg
+console-ports:
+  - name: Serial
+    type: de-9
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+  - name: PSU 2
+    type: iec-60320-c14
```

Comment:
Are these PSU's modular? If so, these should be defined under `module-bays`, and then the associated PSUs should be in module-types if they do not already exist